File: fabric/src/main/java/juuxel/adorn/compat/CinderscapesCompat.java
package juuxel.adorn.compat;

import juuxel.adorn.block.variant.BlockVariant;
import juuxel.adorn.block.variant.PrefixedBlockVariantSet;

import java.util.List;

public final class CinderscapesCompat extends PrefixedBlockVariantSet {
    @Override
    public String getModId() {
        return "cinderscapes";
    }

    @Override
    public List<BlockVariant> getWoodVariants() {
        return createVariants(
            BlockVariant.Wood::new,
            "scorched",
            "umbral"
        );
    }
}

File: fabric/src/main/kotlin/juuxel/adorn/compat/CinderscapesCompat.kt
package juuxel.adorn.compat

import juuxel.adorn.block.variant.BlockVariant
import juuxel.adorn.block.variant.BlockVariantSet

object CinderscapesCompat : BlockVariantSet {
    override val woodVariants = listOf(
        "scorched",
        "umbral",
    ).map { BlockVariant.Wood("cinderscapes/$it") }
}
